api/config_entry_inline_certificate.go

```diff
@@ -12,7 +12,7 @@ type InlineCertificateConfigEntry struct {
 	// Certificate is the public certificate component of an x509 key pair encoded in raw PEM format.
 	Certificate string
 	// PrivateKey is the private key component of an x509 key pair encoded in raw PEM format.
-	PrivateKey string
+	PrivateKey string `alias:"private_key"`
```

Comment:
where does `alias` come from?

Reply:
`alias` is used in the HCL parser we leverage for making it so you can pass the value snake-cased rather than camelized, we leverage it all over the API package, for example:

https://github.com/hashicorp/consul/blob/6599a9be1df8749431ed0f5af7a7a8772ce4281e/api/config_entry.go#L98

It's pretty typical that we prefer snake-casing to camelized values in the ConfigEntry HCL definitions

Reply:
TIL! that's pretty nifty